src/funcCoreTools/getNpmDistTag.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+// tslint:disable-next-line:no-require-imports
+import request = require('request-promise');
+import { ProjectRuntime } from '../constants';
+import { localize } from '../localize';
+
+const npmRegistryUri: string = 'https://aka.ms/W2mvv3';
+
+export interface INpmDistTag { tag: string; value: string; }
+
+export async function getNpmDistTag(runtime: ProjectRuntime): Promise<INpmDistTag> {
+    const tags: { [key: string]: string } = <{ [key: string]: string }>JSON.parse(await <Thenable<string>>request(npmRegistryUri));
+    for (const key of Object.keys(tags)) {
+        if ((runtime === ProjectRuntime.v1 && tags[key].startsWith('1')) ||
```

Comment:
You're probably gonna have to revamp this later...

Reply:
Why? You mean when there's a v3?